source/microbit/microbitspi.cpp

```diff
@@ -57,17 +55,17 @@ STATIC mp_obj_t microbit_spi_init(mp_uint_t n_args, const mp_obj_t *pos_args, mp
         MP_ARRAY_SIZE(allowed_args), allowed_args, (mp_arg_val_t*)&args);
 
     // get pins
-    PinName p_sclk = MICROBIT_PIN_P13;
-    PinName p_mosi = MICROBIT_PIN_P15;
-    PinName p_miso = MICROBIT_PIN_P14;
+    PinName p_sclk = P0_23; // microbit edge pin P13
```

Comment:
We ought to centralise the pin mapping somewhere, if we aren't going to use the DAL headers.
`inc/microbit/microbitpin.h`, perhaps?